src/kernels/MIOpenBatchNormActivBwdPerAct.cl

```diff
@@ -39,6 +39,12 @@
 #define MIOPEN_USE_AMDGCN 1
 #endif
 
+#define MIOPEN_USE_AMDGCN 0
+#if defined(__AMDGCN__) && !MIO_BN_GFX110X
+#undef MIOPEN_USE_AMDGCN
+#define MIOPEN_USE_AMDGCN 1
+#endif
+
```

Comment:
AFAICS here is a logic error that causes `MIOPEN_USE_AMDGCN` to always be 1. Don't add these lines. Instead, replace line 37 with
```C++
#if defined(__AMDGCN__) && !(MIO_BN_GFX103X || MIO_BN_GFX110X)
```

Reply:
Done!